pvlib/clearsky/nrel.py

```diff
@@ -0,0 +1,187 @@
+"""
+Richard E. Bird 
+        Clear Sky Broadband
+        Solar Radiation Model 
+From the publication "A Simplified Clear Sky model for Direct and Diffuse
+Insolation on Horizontal Surfaces" by R.E. Bird and R.L Hulstrom, SERI Technical
+Report SERI/TR-642-761, Feb 1991. Solar Energy Research Institute, Golden, CO.
+
+The model is based on comparisons with results from rigourous radiative transfer
```

Comment:
text needs to be edited so that it's not specific to the excel file. Not sure if there is a licensing/attribution issue here.